libs/core/src/lib/multi-input/multi-input.component.html

```diff
@@ -83,9 +87,13 @@
                 [attr.aria-required]="required"
                 [class.fd-input--compact]="compact"
                 (ngModelChange)="_handleSearchTermChange($event)"
+                [attr.aria-haspopup]="!mobile"
+                [attr.aria-expanded]="!!(open && !mobile && displayedValues && displayedValues.length)"
                 (keydown)="_handleInputKeydown($event)"
                 [ngModelOptions]="{ standalone: true }"
                 [attr.id]="inputId"
+                [attr.aria-describedby]="tokenHiddenId"
+                aria-roledescription="Multi Value Input"
```

Comment:
Same as above

Reply:
fixed

Reply:
We're not using `@angular/localize` in `core`, use `@Input()` instead for passing aria labels.

Reply:
Please use `@Input()` for `aria-roledescription="Multi Value Input"` so the customers can pass own `aria-roledescription` value.